Another clean up for global.mtls.enable (#6611)
istio/istio.io: content/en/docs/tasks/security/citadel-config/plugin-ca-cert/index.md

## Before you begin

1. Follow the [Istio installation guide](/docs/setup/install/istioctl/) to install Istio.

> 1. Enable mutual TLS in STRICT mode as show in [authentication policy](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode). This step is optional as the default mode (`PERMISSIVE`) already supports mutual TLS.

Reviewer: ```suggestion
1. **Optional:** Enable mutual TLS in `STRICT` mode as show in the [authentication policy task](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode). The default `PERMISSIVE` mode already supports mutual TLS traffic.
```

        --from-file=samples/certs/cert-chain.pem
    {{< /text >}}

> 1.  Redeploy Citadel with `security.selfSigned` to `false`.

Reviewer: Is this correct for 1.5? How is this working with Istiod?

Author: https://github.com/istio/istio.io/pull/6644/files#diff-8 taking care of this, so I only remove global.mtls.enabled in this one for consistency.

> 1.  Redeploy Citadel with `security.selfSigned` to `false`.

Reviewer: ```suggestion
1.  Redeploy Citadel with `security.selfSigned`set to `false`.
```
